test/Microsoft.AspNetCore.Routing.Tests/LinkGeneratorRouteValuesAddressExtensionsTest.cs

```diff
@@ -154,15 +154,17 @@ public void GetUriByRouteValues_WithHttpContext_WithPathBaseAndFragment()
             var linkGenerator = CreateLinkGenerator(endpoint1, endpoint2);
 
             var httpContext = CreateHttpContext();
-            httpContext.Request.Scheme = "http";
-            httpContext.Request.Host = new HostString("example.com");
+            httpContext.Request.Scheme = "httsp";
```

Comment:
`httsp`? 🤔

Reply:
httsp is HTTP 3 - I'm not much public information is available yet 😆

Reply:
I'm going to leave this here for posterity. This value doesn't show up in the test output, it's specifically there to demonstrate that it doesn't get used.